buildkitd/Earthfile

```diff
@@ -12,7 +12,7 @@ buildkitd:
             ARG BUILDKIT_BASE_IMAGE=$BUILDKIT_PROJECT+build
         END
     ELSE
-        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:9c256c19c5397aee1ba0379590d685c2e87ff17c+build
+        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:53d3f36ec9cd31894bd16fa2145b9436031c3b50+build
```

Comment:
can you coordinate with @r-LaForge regarding #2285

I suspect you're buidkit PR will need to be rebased against the ongoing buildkit-fork-update PR.

Reply:
I just did a rebase over there; I thought it was merged already :( I can wait, its no problem